### PR TITLE
Fix dev version

### LINKED ISSUE #1597: hyperspy dev version number not supported by pip
With hyperspy dev installed, installing hyperspyui by `pip install hyperspyui` triggers a new install of `hyperspy-1.2+1` from PyPI.

### PR DESCRIPTION
**Pull request by francisco-dlp**
Fixes #1597

### CODE REVIEW

**Comment by thomasaarholt** [comment]
woohoo

**Comment by thomasaarholt** [comment]
Error:
```python
~\hyperspy [master ≡ +4 ~0 -0 !]> git checkout pr/1630
Branch pr/1630 set up to track remote branch pr/1630 from upstream.
Switched to a new branch 'pr/1630'
rm: cannot remove 'C:/Users/thomasaar/hyperspy/hyperspy/io_plugins/unbcf_fast.c*': No such file or directory
rm: cannot remove 'C:/Users/thomasaar/hyperspy/hyperspy/io_plugins/unbcf_fast.cpython-*.pyd': No such file or directory
Traceback (most recent call last):
  File "C:/Users/thomasaar/hyperspy/setup.py", line 47, in <module>
    import hyperspy.Release as Release
  File "C:\Users\thomasaar\hyperspy\hyperspy\__init__.py", line 28, in <module>
    from . import Release
  File "C:\Users\thomasaar\hyperspy\hyperspy\Release.py", line 28
    version = "1.3.dev
```

File: hyperspy/Release.py [hunk 1]
```diff
@@ -25,7 +25,7 @@
 # When running setup.py the "+dev" string will be replaced (if possible)
 # by the output of "git describe" if git is available or the git
 # hash if .git is present.
-version = "1.3+dev"
+version = "1.3.dev
```
Comment (thomasaarholt) [inline hyperspy/Release.py:28]: Needs a final "

**Comment by thomasaarholt** [comment]
Hm, I fixed it on my my computer, but I'm unsure how to test it. 

I checkout your PR, fix it, checkout again.
Then I tried uninstalling hyperspyUI and then installing it from pip, but then it doesn't recognise the PR hyperspy and tries installing 1.1.1.

But now I tried editing the Release.py version number to be 1.3, checkout again, and it STILL doesn't work...

**Comment by thomasaarholt** [comment]
Hm. Reading [PEP 440](https://www.python.org/dev/peps/pep-0440/) now, I see the following:
> This change is designed to ensure that an integrator provided version like `pip 1.5+1` or `pip 1.5+1.git.abc123de` will still satisfy a version specifier like `pip>=1.5` .

Now I'm confused. Why is my hyperspy master not recognised by pip?

**Comment by francisco-dlp** [comment]
Did you try uninstalling hyperspy and installing again from a tar.gz created from this branch? It works for me.

I'll merge it as it is an improvement and it only affects developers, so we can change it again at anytime without needing to release.

**Comment by francisco-dlp** [comment]
Thanks for testing!

**Pull request merged**